tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -1886,6 +1886,48 @@ namespace unwrapped_begin_end {
     }
 } // namespace unwrapped_begin_end
 
+namespace closure {
+    // Verify that range adaptor closure capture with the proper value category
+
+    enum class path { none, lvalue, const_lvalue, rvalue, const_rvalue };
+
+    template <path allowed>

```

Comment:
Naming nitpick: I recommend capitalizing the template parameter `allowed` - I recently fixed all occurrences of lowercase template parameters in test code (aside from libcxx-derived sources).